modules/command/publish/send-assertion-command.js

```diff
@@ -31,7 +31,7 @@ class SendAssertionCommand extends Command {
 
         for (const node of nodes) {
             try {
-                await this.publishService.store({ rdf, id: assertion.id }, node);
+                this.publishService.store({ rdf, id: assertion.id }, node);
```

Comment:
This can cause a problem if store throws exception. Since store method call returns a promise we should handle error differently (not with try/catch).
Implementation should be something like this:
this.publishService.store(...).catch(...)